flow/project.py

```diff
@@ -2144,6 +2697,11 @@ def submit(self, bundle_size=1, jobs=None, names=None, num=None, parallel=False,
             The default is :py:class:`IgnoreConditions.NONE`.
         :type ignore_conditions:
             :py:class:`~.IgnoreConditions`
+        :param ignore_conditions_on_submit:
```

Comment:
The name of this parameter appears to be a misnomer. Since the documentation implies that this used to ignore conditions on execution *after* submission, correct?

Reply:
I thought the name implied that upon submission conditions would not be checked, not at submission, but I could change it to `ignore_conditions_on_execution`

Reply:
Not sure what the best name is but would be best to find one that is non-ambiguous.